theano/tensor/signal/pool.py

```diff
@@ -708,6 +717,10 @@ def c_code(self, node, name, inp, out, sub):
         ds0, ds1 = self.ds
         st0, st1 = self.st
         pd0, pd1 = self.padding
+        if self.openmp:
```

Comment:
You must modify the \__init__ of this class. You changed PoolGrad.\__init__, but MaxPoolGrad.\__init__ must also be changed